### PR TITLE
Allowed custom sampling tolerance

### PR DESCRIPTION
**Pull request by jlstevens**
The tolerance for all types is now consistent with datetimes, you can now specify a global tolerance value using ``hv.config`` and tolerance values per ``Image`` by specifying ``rtol`` in the constructor.

TODO:

- [x] Add unit tests.
- [x] Add docs.

### CODE REVIEW

File: holoviews/core/util.py [hunk 1]
```diff
@@ -73,6 +73,11 @@ class Config(param.ParameterizedFunction):
        recommended that users switch this on to update any uses of
        __call__ as it will be deprecated in future.""")
 
+    rtol = param.Number(default=10e-6, doc="""
```
Comment (philippjfr) [inline holoviews/core/util.py]: The config option should probably have a more specific name. Also it's not for all gridded data but specifically for Images.

Comment (jlstevens) [reply holoviews/core/util.py]: Sure. ``image_rtol`` then? Or do you have an alternative suggestion?

**Comment by jbednar** [comment]
Looks good, but ``image_rtol`` seems very obscure.  Maybe ``sampling_tolerance`` or something more readable like that?

**Comment by philippjfr** [comment]
>Maybe sampling_tolerance or something more readable like that?

Sampling tolerance of what though, this is an option on the global config, so naming it something non-specific makes it seem like this will be used globally for all kinds of things.

**Comment by jlstevens** [comment]
Ready for final review/merge when the tests go green.

**Comment by philippjfr** [comment]
Looks good, tests are passing and coveralls seems stuck.

**Pull request merged**